src/apps/popup/pages/transfer/confirm-step.tsx

```diff
@@ -134,7 +134,7 @@ export const ConfirmStep = ({
               {listItems.text}
             </Typography>
             <AmountContainer>
-              <Typography type="captionHash">{`${listItems.amount} ${listItems.symbol}`}</Typography>
+              <Typography type="captionHash">{`${listItems.amount} ${listItems?.symbol}`}</Typography>
```

Comment:
if listItems will not have a symbol value it will return undefined and you will get text `10 undefined`